src/twisted/web/_stan.py

```diff
@@ -210,13 +214,13 @@ def _clone(self, obj, deep):
         @return: a clone of C{obj}.
         """
         if hasattr(obj, "clone"):
-            return obj.clone(deep)
+            return obj.clone(deep)  # type: ignore[attr-defined, no-any-return]
```

Comment:
We probably should `cast` instead of `no-any-return`, I think.

Reply:
Agreed.